extension/ui/components/markdown/UrlMentionBlock.tsx

```diff
@@ -0,0 +1,21 @@
+import { Chip } from "@dust-tt/sparkle";
+import { visit } from "unist-util-visit";
+
+export function UrlMentionBlock({ title }: { title: string; url: string }) {
```

Comment:
Note: "url" is unused yet, but will modify the `Chip` component to make it clickable

Reply:
:+1: